workitem/field_definition.go

```diff
@@ -72,9 +72,15 @@ type FieldType interface {
 	ConvertFromModel(value interface{}) (interface{}, error)
 	// Implement the Equaler interface
 	Equal(u convert.Equaler) bool
-	// DefaultValue is called if a field is not specified. In it's simplest form
-	// the DefaultValue returns the given input value without any conversion.
-	DefaultValue(value interface{}) (interface{}, error)
+	// GetDefaultValue is called if a field is not specified. In it's simplest
```

Comment:
>GetDefaultValue is called if a field is not specified

@kwk is this sentence incomplete? It doesn't make sense to me.

Reply:
@jarifibrahim No, the sentence is complete. I meant: "If a field is not specified (inside a work item create payload)."